frontend-v2/src/features/data/normaliseDataHeaders.ts

```diff
@@ -63,19 +70,26 @@ const normalisation = {
     "obs_unit",
     "obsunit",
     "units_conc",
+    "unit_observation", 
+    "unit observation", 
+    "concentration unit", 
+    "concentration_unit", 
+    "unit concentration", 
+    "unit_concentration",
   ],
   "Observation ID": ["observation id", "observation_id", "ytype", "dvid"],
   "Observation Variable": ["observation variable", "observation_var"],
   Occasion: ["occasion", "occ"],
   Regressor: ["x", "regressor"],
-  Time: ["time", "t", "ivar"],
+  Time: ["time", "t", "ivar", "Hour_Actual"],
```

Comment:
```suggestion
  Time: ["time", "t", "ivar", "hour_actual"],
```

The CSV column headers are converted to lowercase before being compared with these arrays, so I've got into the habit of using lowercase as the default.